contracts/common.yml

```diff
@@ -9863,6 +9866,14 @@ components:
       enum:
         - implicit
         - explicit
+    ColorMapping:
+      type: object
+      description: maps series id to the color index that is used to index into the colors array
```

Comment:
I think this is a bit of an obvious comment. I think a better description would explain what a color mapping is so that someone reading this definition who wasn't familiar with color mapping would understand what it's for.

>A color mapping is an object that maps time series data to a UI color scheme to allow the UI to render graphs consistent colors across reloads.